traffic_ops/traffic_ops_golang/traffic_ops_golang.go

```diff
@@ -109,7 +109,7 @@ func main() {
 		WriteTimeout:      time.Duration(cfg.WriteTimeout) * time.Second,
 		IdleTimeout:       time.Duration(cfg.IdleTimeout) * time.Second,
 	}
-
+	server.SetKeepAlivesEnabled(true)
```

Comment:
Keepalives are enabled by default https://golang.org/pkg/net/http/#Server.SetKeepAlivesEnabled

This shouldn't be necessary

Reply:
This was cruft from testing that wasn't meant for this PR. Will remove it.